rca/landingpages/models.py

```diff
@@ -560,9 +562,61 @@ class Meta:
         verbose_name = "Landing Page - Enterprise"
 
 
+class EELandingPageRelatedEditorialPage(RelatedPage):
+    source_page = ParentalKey(
+        "landingpages.EELandingPage", related_name="related_editorial_pages"
+    )
+    panels = [PageChooserPanel("page", ["editorial.EditorialPage"])]
+
+
 class EELandingPage(BasePage):
     template = "patterns/pages/editorial_event_landing/editorial_event_landing.html"
     max_count = 1
 
+    news_link_text = models.TextField(
+        max_length=120,
+        blank=False,
+        null=True,
+        help_text=_("The text do display for the link"),
+    )
+    news_link_target_url = models.URLField(blank=False, null=True)
+
     class Meta:
         verbose_name = "Landing Page - Editorial and Events"
+
+    @property
+    def news(self):
+        news = []
+        picked_news = self.related_editorial_pages.first().page.specific
```

Comment:
This will throw a value error if `related_editorial_pages` was ever empty. I know its required but the linked page could be deleted (possibly an edge case, but if it can happen it probably will!)